### PR TITLE
Fix pithos-app update

### PR DESCRIPTION
**Pull request by sofuture**
- [x] fix update
 - [x] support rollback to arbitrary previous state

### CODE REVIEW

File: resources/rename-cassandra.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
```
Comment (UnderGreen) [inline resources/rename-cassandra.yaml]: Why in update tasks we use DS instead of Job?

**Comment by UnderGreen** [comment]
LGTM, except of my question about DaemonSet.

File: resources/rename-cassandra.yaml (hunk 1 above)
Comment (sofuture) [inline resources/rename-cassandra.yaml]: It has to run on each node :cry: There's no way to do that with a job.

Comment (sofuture) [inline resources/rename-cassandra.yaml]: Well, you *could* create a job per node, which is an alright idea.

**Pull request merged**